src/app/core/admin/admin-entity-details/admin-entity-field/admin-entity-field.component.html

```diff
@@ -79,7 +86,16 @@ <h2 mat-dialog-title i18n>Configure Field "{{ entitySchemaField.label }}"</h2>
             </mat-form-field>
 
             <mat-form-field>
-              <mat-label i18n>Type</mat-label>
+              <mat-label>
+                <span i18n>Type</span>
+                @if (publicFormConfig) {
+                  <fa-icon
+                    icon="question-circle"
+                    matTooltip="The type of the field cannot be changed here because these changes only apply to this form. For data consistency, the field type is disabled."
```

Comment:
```suggestion
                    matTooltip="The type of the field cannot be changed here because these changes only apply to this form. For data consistency, you have to change the type for the field overall."
```